app/middlewares/signUpValidations.js

```diff
@@ -0,0 +1,22 @@
+const { check, validationResult } = require('express-validator/check'),
+  { validateUser } = require('./validations'),
+  error = require('../errors');
+
+exports.signUpValidate = (req, res, next) => {
```

Comment:
This function should be generic for avoid repeat code in the future

Reply:
I do not understand, is the function incorrect or the name?

Reply:
The function not the name, should be more generic

Reply:
Change name too